src/lib/server/models/discord/attachment/index.ts

```diff
@@ -0,0 +1,17 @@
+import { type InferOutput, nullable, number, object, optional, string } from 'valibot';
+import { Snowflake } from '$lib/server/models/discord/snowflake';
+
+export const Attachment = object({
+    id: optional(Snowflake),
```

Comment:
Isn't this a required field?
```suggestion
    id: Snowflake,
```